src/utils/rfqm_health_check.ts

```diff
@@ -1,8 +1,18 @@
+import { BigNumber } from '@0x/utils';
```

Comment:
I think the standard file naming in utils is to have a `_utils` suffix

Reply:
yeahhhh.. there's a few exceptions. I thought about this and I really don't consider this a "util" at all, but we don't really have a place for this type of file. We should think on this.